services/production-tools/values-idfint.yaml

```diff
@@ -0,0 +1,13 @@
+production-tools:
+  butler_secret_path: "secret/k8s_operator/data-int.lsst.cloud/butler-secret"
+  environment:
+    BUTLER_URI: "s3://butler-us-central1-panda-dev/dc2/butler-external.yaml"
+    PGPASSFILE: "/home/worker/.lsst/postgres-credentials.txt"
+    AWS_SHARED_CREDENTIALS_FILE: "/home/worker/.lsst/aws-credentials.ini"
+    S3_ENDPOINT_URL: "https://storage.googleapis.com"
```

Comment:
None of these vary by environment and should therefore be hard-coded in the deployment so that you don't have to repeat them in every values file.